### ISSUE TITLE
Bring back tests for parsing rustc own tests

### ISSUE THREAD

**Issue by matklad** [intellij-rust/intellij-rust#1330]
We used to parse all rustc tests as a part of our own test suite, but we no longer do this, because rustup no longer ships test with rustc source code. We need to review these tests by manually cloning rustc reposotiry during tests.

**Comment by farodin91** [comment]
https://github.com/rust-lang-nursery/rustup.rs/blob/master/README.md#can-rustup-download-the-rust-source-code

**Comment by matklad** [comment]
Yeah, this is exactly how we used to do those tests, but as of recently rustup downloads the sources only of standard library, and does not download sources of the compiler or it's tests. 

And it s the tests we are most interested in.
